src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMen = people.filter(person =>
+    (century
+      ? Math.ceil(person.died / 100) === century
+      : true
+    )
+    && person.sex.toLowerCase() === 'm');
+
+  const mensAges = filterMen.map(men => men.died - men.born);
+  const sumAges = mensAges.reduce((sum, year) => sum + year, 0);
+
+  return sumAges / mensAges.length;
```

Comment:
You calculate the average age in each function. It would be cool to put this operation in a separate function and use it instead of writing the same thing several times.